### DISCUSSION TITLE
Label med tooltip

### DISCUSSION THREAD

**Discussion by Stormoen** [fremtind/jokul#3356] [Annet]
Kjøpsløsningene på PM har ofte behov for label med tooltip i skjema. Kunne den kanskje vært lagt til i komponenten, tooltip true/false?

![Skjermbilde 2023-01-06 kl  13 12 37](https://user-images.githubusercontent.com/64519175/211010978-ce73fd09-574b-408b-a99e-8e4f3b70f5c6.png)

**Comment by wkillerud** [comment]
Vi jobber med saken i #3227. Øyvind har en Figma-branch [lenket i issuet](https://github.com/fremtind/jokul/issues/3227#issuecomment-1372345864) som du gjerne må sjekke og gi tilbakemelding på @Stormoen.

**Reply by Stormoen** [reply]
Supert. Da vet jeg desverre ikke hvordan jeg lukker / sletter denne saken, haha. Hjelp?

**Reply by wkillerud** [reply]
Ingen fare, den kan bare ligge 😄 Diskusjonstråder kan bare låses eller slettes.

**Comment by onordbo** [comment]
@Stormoen: Dette skal nå være mulig per #3227. Si fra om det ikke funker, så hjelper jeg til 😊
